src/features/InteractWithContracts/components/Interact.tsx

```diff
@@ -390,7 +391,12 @@ function Interact(props: CombinedProps) {
             </FieldWrapper>
 
             <ButtonWrapper>
-              <Button color={WHITE} disabled={wasContractInteracted} onClick={submitInteract}>
+              <Button
+                color={WHITE}
+                disabled={wasContractInteracted}
+                onClick={submitInteract}
+                fullwidth={isMobile}
```

Comment:
Is it only intended to be full width on mobile?

Reply:
Yes it's only fullwidth on mobile :)